src/main/java/com/fp3/haras/model/Produto.java

```diff
@@ -1,12 +1,13 @@
 package com.fp3.haras.model;
 
+import com.fp3.haras.utils.EntityUtils;
 import javax.persistence.Entity;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
 import javax.persistence.Table;
 
-@Entity
+@Entity(name="Produto")
```

Comment:
When interacting with the database I was using FROM "table name" instead of entity name, so I decided to name the entities and refer to them instead, but after it worked, I noticed that I didn't need to do that nomination lol